project/Dependencies.scala

```diff
@@ -20,8 +18,13 @@ object Dependencies {
 
   val PekkoActorVersion = "1.1.1"
   val PekkoHttpVersion  = "1.0.1"
-  val JenaVersion       = "5.1.0"
-  val Rdf4jVersion      = "5.0.2"
+
+  // rdf and graph libraries
+  // topbraid/shacl is not yet compatible with jena 5 so we need to use jena 4 for now
+  // see: https://github.com/TopQuadrant/shacl/pull/177
+  val JenaVersion          = "4.10.0"
+  val Rdf4jVersion         = "5.0.2"
+  val TopbraidShaclVersion = "1.4.3"
```

Comment:
I don't see the reason to create version constant if it's used only once.
Also the compatibility comment is already added to `scala-steward.conf` which should be enough.